lumicks/pylake/channel.py

```diff
@@ -131,6 +131,32 @@ def downsampled_over(self, range_list, reduce=np.mean, where='center'):
 
         return Slice(TimeSeries(d, t), self.labels)
 
+    def downsampled_to(self, frequency, reduce=np.mean, where='center'):
+        """Return a copy of this slice which is downsampled by `factor`
+
+        Parameters
+        ----------
+        frequency : int
+            The desired downsampled frequency downsampled (per second)
+        reduce : callable
+            The `numpy` function which is going to reduce multiple samples into one.
+            The default is `np.mean`, but `np.sum` could also be appropriate for some
+            cases, e.g. photon counts.
+        where : str
+            Where to put the final time point.
+            'center' time point is put at start + stop / 2
+            'left' time point is put at start
```

Comment:
Maybe a small warning about non-integer ratios not giving an equal number of samples per resulting sample may be in order?